tests/glade_tests.sh

```diff
@@ -2,13 +2,13 @@
 
 # If $top_srcdir has not been set by automake, import the test environment
 if [ -z "$top_srcdir" ]; then
-    top_srcdir="$(realpath $(dirname "$0")/..)"
-    . ${top_srcdir}/tests/testenv.sh
+    top_srcdir="$(realpath "$(dirname "$0")/..")"
```

Comment:
Are you sure that this works as expected. The parser will probably not work correctly on all these `"`. Also the original variant is wrong too. I guess the correct way to fix this should be to start escaping the inner quotes.

Reply:
Although the question is if the `$(..)` sections are not parsed separately.

Reply:
Yes, it works. As you already guessed, $(...) is another "scope" - it creates a subshell where the quotes are parsed independently of the surroundings. That leads to some really bizarre things like this one here.

Reply:
Thanks for verification of this.